web/src/features/fba/components/map/FBAMap.tsx

```diff
@@ -284,6 +287,14 @@ const FBAMap = (props: FBAMapProps) => {
     })
     mapObject.setTarget(mapRef.current)
 
+    const scaleBar = new ScaleLine({
+      bar: true,
+      minWidth: 160,
+      steps: 5
```

Comment:
Looks good! Would changing this to 4 solve the overlap issue?

Reply:
Great suggestion! It fixed the problem by reducing the amount of text. Thanks!